test/Microsoft.DotNet.Cli.Utils.Tests/GivenAProjectToolsCommandResolver.cs

```diff
@@ -456,5 +457,12 @@ private ProjectToolsCommandResolver SetupProjectToolsCommandResolver()
 
             return projectToolsCommandResolver;
         }
+
+        private string GetToolDepsJsonGeneratorProject()
+        {
+            //  When using the product, the ToolDepsJsonGeneratorProject property is used to get this path, but for testing
+            //  we'll hard code the path inside the SDK since we don't have a project to evaluate here
+            return Path.Combine(new RepoDirectoriesProvider().Stage2Sdk, @"Sdks\Microsoft.NET.Sdk\build\GenerateDeps\GenerateDeps.proj");
```

Comment:
Can't use slashes across platforms.  Need to use `Path.Combine("one", "two", "three")`